src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php

```diff
@@ -32,14 +32,14 @@ class AppKernel extends Kernel
 
     public function __construct($varDir, $testCase, $rootConfig, $environment, $debug)
     {
-        if (!is_dir(__DIR__ . '/' . $testCase)) {
+        if (!is_dir(__DIR__.'AppKernel.php/'. $testCase)) {
```

Comment:
Why did you these (also the one below) changes?

```suggestion
        if (!is_dir(__DIR__.'/'.$testCase)) {
```